src/SigSpec.CodeGeneration/Models/ReturnTypeModel.cs

```diff
@@ -0,0 +1,24 @@
+﻿using NJsonSchema;
+using NJsonSchema.CodeGeneration;
+using SigSpec.Core;
+
+namespace SigSpec.CodeGeneration.Models
+{
+    public class ReturnTypeModel
+    {
+        private readonly string _name;
+        private readonly JsonSchema4 _parameter;
+        private readonly TypeResolverBase _resolver;
+
+        public ReturnTypeModel(string name, JsonSchema4 parameter, TypeResolverBase resolver)
+        {
+            _name = name;
+            _parameter = parameter;
+            _resolver = resolver;
+        }
+
+        public string Name => _name;
+
+        public string Type => _resolver.Resolve(_parameter.ActualTypeSchema, _parameter.IsNullable(SchemaType.JsonSchema), _name);
```

Comment:
We should change all model properties from `Type => ...` to `Type { get; } = ...` to avoid multiple evaluations and thus improved render performance... what do you think?

Reply:
What do you think about this? https://github.com/RSuter/SigSpec/pull/4/commits/12a762a5691205f46895573caf33e59412202565

Reply:
Good to use lazy, but TBH here I think it's just overhead and not really needed because the property is always used... I'd just do it sync in the ctor - it's the easiest way to understand it and requires the least amount of code.

Reply:
Fine by me. I've removed the use of Lazy.